### ISSUE TITLE
Generated imports include the .ts extension

### ISSUE THREAD

**Issue by OrKoN** [prisma/graphqlgen#155]
## Description
Generated imports include the .ts extension

## Steps to reproduce
1. generate a simple model
2. inspect generated files

## Expected results

```
import { ModelResolvers } from './src/generated/graphqlgen';
```

## Actual results
```
import { ModelResolvers } from './src/generated/graphqlgen.ts';
```

Giving a TS error: `TS2691: An import path cannot end with a '.ts' extension. Consider importing './src/generated/graphqlgen' instead.`
## Versions

TypeScript version:

```
3.0.3
```

TypeScript config:

```
{
  "compilerOptions": {
    "module": "commonjs",
    "target": "ES2017",
    "strict": true,
    "noImplicitReturns": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "sourceMap": true,
    "skipLibCheck": false,
    "outDir": "./dist",
    "moduleResolution": "node",
    "lib": ["esnext"]
  },
  "include": ["src/**/*.ts"],
  "exclude": ["node_modules", "./**/flycheck*.ts"]
}
```

**Comment by OrKoN** [comment]
And still the import path is not correct:

```
language: typescript

schema: ./src/schema.graphql
context: ./src/app/domain/RequestContext.ts:RequestContext
models:
  ServiceInformation: ./src/app/domain/ServiceInformationResponse.ts:ServiceInformationResponse

output: ./src/generated/graphqlgen.ts

resolver-scaffolding:
  output: ./src/generated/tmp-resolvers/
  layout: single-file
```

Generates resolvers which try to relatively import form ` './src/generated/graphqlgen';` while they are themselves in `./src/generated/tmp-resolvers`

**Comment by kevinmarrec** [comment]
Duplicate of #117, fixed by PR #140, but it hasn't been published on npm yet.

**Comment by schickling** [comment]
Published now 🎉
